src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -2656,8 +2690,22 @@ public Bundle tProcedure2Procedure(org.openhealthtools.mdht.uml.cda.Procedure cd
 		}
 
 		// code -> code
-		if(cdaProcedure.getCode() != null && !cdaProcedure.getCode().isSetNullFlavor()) {
-			fhirProc.setCode(dtt.tCD2CodeableConcept(cdaProcedure.getCode()));
+		CD code = cdaProcedure.getCode();
```

Comment:
lgtm but maybe this can all go into tCD2CodeableConcept at some point?

Reply:
meaning 2694 -> 2708

Reply:
I was going to do that after structuring some code but fair. Implemented now.